### PR TITLE
Stick mode now in one global variable, it should work in Horus emerge…

### PR DESCRIPTION
**Pull request by projectkk2glider**
…ncy mode

### CODE REVIEW

File: radio/src/opentx.h [hunk 1]
```diff
@@ -1486,57 +1484,60 @@ extern void opentxInit();
...
-    } version;
+  struct
+  {
+    char id[27];
+  } version;
 #endif
+  struct
+  {
+    uint8_t stickMode;
+  } modelSettings;
```
Comment (projectkk2glider) [inline radio/src/opentx.h]: The only real change to this file is new struct `modelSettings`, the rest are just cosmetics.

File: radio/src/gui/taranis/menu_general_setup.cpp [hunk 2]
```diff
@@ -473,9 +477,9 @@ void menuGeneralSetup(uint8_t event)
         if (attr && s_editMode>0) {
           CHECK_INCDEC_GENVAR(event, g_eeGeneral.stickMode, 0, 3);
```
Comment (bsongis) [inline radio/src/gui/taranis/menu_general_setup.cpp]: The stick mode will change without pressing [Enter], I would prefer we change the reusable variable here

Comment (bsongis) [inline radio/src/gui/taranis/menu_general_setup.cpp]: I remember one issue we solved in the past with this code, somebody going from mode1 to mode3 with the model ON. Perhaps @romoloman will remember the details, I discussed this with him at the time

File: radio/src/opentx.h (hunk 1 above)
Comment (bsongis) [inline radio/src/opentx.h]: It should be called generalSettings, no?

**Comment by bsongis** [comment]
Yes except the small problems I nociced, this is a better implementation than mine

**Comment by bsongis** [comment]
Thanks!

**Pull request merged**